docker/.env

```
@@ -208,6 +208,8 @@ UV_CUBE_AGENT_PORT=8901
 UV_CUBE_AGENT_SERVER_CERT=
 UV_CUBE_AGENT_SERVER_KEY=
 UV_CUBE_AGENT_INSTANCE_ID=
+UV_CUBE_GUARDRAILS_URL=http://guardrails:8001
+UV_CUBE_AGENT_USE_GUARDRAILS_BACKEND=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The UV_CUBE_AGENT_USE_GUARDRAILS_BACKEND key should go before the UV_CUBE_GUARDRAILS_URL key

(UnorderedKey)